packages/core/src/common/uri.ts

```diff
@@ -14,9 +14,12 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
+import { setUriThrowOnMissingScheme } from 'vscode-uri';
 import Uri from 'vscode-uri';
 import { Path } from './path';
 
+setUriThrowOnMissingScheme(false);
```

Comment:
We have to disable it because of plugin ext: https://github.com/theia-ide/theia/issues/4487

cc @evidolob

Reply:
Please put your comment into the code

Reply:
done